src/worker/startup.ts

```diff
@@ -154,15 +163,23 @@ export async function stop() {
 	// Stop sending metrics for queues
 	clearInterval(timer);
 
-	// Close all queues
+	// Pauses message processing in all queues globally.
+	// The first parameter of this function "isLocal" controls if all queues will be paused
+	await queues.discovery.pause(false, true)
```

Comment:
Why are we not doing same for other queues (e.g. installation)?

Reply:
Poor and misleading bull queues documentation.

Should have done it for all the queues. But it doesn't work for some reason at all.